library/foundations/generator-config.ts

```diff
@@ -0,0 +1,5 @@
+/** Customizable Configuration for the Generator */
```

Comment:
This config can be reused to configure the generator based on project needs, e.g. as an alternative to special tokens